merlin/models/utils/dataset.py

```diff
@@ -49,6 +50,18 @@ def dataset_to_coo(dataset: Dataset):
     return coo_matrix((targets.astype("float32"), (userids, itemids)))
 
 
+def unique_by_tag(dataset: Dataset, tag: Union[str, Tags], id_tag: Union[str, Tags]):
```

Comment:
What about renaming to something more intuitive like `unique_rows_by_features(dataset, features_tag, grouping_tag)` ?